From af43d6cb00606dbdbd4870e715916b345086b3ef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Wed, 13 Oct 2010 19:39:14 +0200
Subject: [PATCH] suppress modprobe errors on builtins (credits to Kay Sievers)

Install /lib/modules/$kv/modules.builtin.bin to suppress modprobe error
messages saying module was not found, while it's built-in.

Credits go to Kay Sievers who enlighten us about meaning of this cool
file.
---
 modules.d/90kernel-modules/install |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/modules.d/90kernel-modules/install b/modules.d/90kernel-modules/install
index 2f19d0c..05819f5 100755
--- a/modules.d/90kernel-modules/install
+++ b/modules.d/90kernel-modules/install
@@ -2,3 +2,4 @@
 [ -f /etc/modprobe.conf ] && dracut_install /etc/modprobe.conf
 dracut_install $(find /etc/modprobe.d/ -type f -name '*.conf')
 inst_hook cmdline 01 "$moddir/parse-kernel.sh"
+inst "$srcmods/modules.builtin.bin"
